plugins/node/opentelemetry-instrumentation-pg/src/internal-types.ts

```diff
@@ -57,6 +57,7 @@ export interface PgPoolOptionsParams {
 
 export interface PgPoolExtended extends pgPoolTypes<pgTypes.Client> {
   options: PgPoolOptionsParams;
+  eventListenersSet: boolean; // flag to identify if the event listeners for instrumentation have been set
```

Comment:
This is not a property that actually exists so we'd be defining it as we go - I think to avoid any problems downstream with adding an enumerable property we should a Symbol as a key like we do in other instrumentations, which won't be enumerable and should be a bit safer for the user.

See [here](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/6dfe93c01c6a5750d69045a1ae037b8712857b8f/plugins/node/opentelemetry-instrumentation-aws-sdk/src/aws-sdk.ts#L70-L73) for an example.

Reply:
Done